src/server/utils/pollModuleMap.js

```diff
@@ -23,11 +23,6 @@ import {
 
   holocron as holocronMetrics,
 } from '../metrics';
-import {
-  getModulesUsingExternals,
-  setModulesUsingExternals,
-} from './onModuleLoad';
-import { backupModuleStateConfig, restoreModuleStateConfig } from './stateConfig';
```

Comment:
was this not being used?

Reply:
modules are either accepted or rejected, we don't have a situation anymore where we revert a partially loaded module map